### ISSUE TITLE
Issues under multiple Bodies does not show up in Dashboard

### ISSUE THREAD

**Issue by leowmjw** [mysociety/fixmystreet#1342]
I have setup Category as per #1323; and it works correctly and is less confusing.  However it seems to have messed with the Reporting and Dashboard; please give guidance on the scenario below.

Body hierarchy has been configured (via the parent Body drop-down) as below:
Majlis Bandaraya Petaling Jaya -> MBPJ Zon 3 -> PPR Kota Damansara (most specific)

An example scenario:
https://aduanku.my/report/857

In the internal Report, it shows the case correctly being assigned to all 3 bodies under the area:
```
Body: MBPJ Zon 3, Majlis Bandaraya Petaling Jaya, PPR Kota Damansara (other areas: 1,13,51,6792,7206,7398,7403).  
```
Also, the email for the Bodies is received.

However, when logging in to Dashboard (in URL /dashboard) using a User assigned to the Body (I tried each of the Body in the hierarchy) but it does not appear anywhere.

**Expectation:**
1) The issue should appear under the Dashboard of the most specific Body (or if not feasible, it should appear in all the Bodies where it falls under).

2) The issue should appear under the reports page of all the Body in the hierarchy.  It appears in the first two level in the hierarchy, but it DOES NOT shows up in the most specific one --> https://aduanku.my/reports/PPR+Kota+Damansara

3) Report number should appear in the most specific body (in this case PPR Kota Damansara) but the number should "roll up" to the parents MBPJ Zon 3, Majlis Bandaraya Petaling Jaya but the numbers does not seem to match up.

**Comment by dracos** [comment]
Version 1.7 of FixMyStreet fixed some bugs in the dashboard (I don't think it worked in places with more than one body for a report before then), and in general to do with multiple body handling (I think your expectation 2 was fixed, it would only show a report in a maximum of two bodies), so if you're running an older version, upgrading will hopefully help. I hope to do a new release very soon, perhaps tomorrow, which includes multiple photo upload support and optional Facebook login, if you want to wait for that.

**Comment by dracos** [comment]
Version 1.8 was released as above, but as I say 1.7 or above should fix this; please reopen if it doesn't.
